siliconcompiler/tools/openroad/scripts/sc_floorplan.tcl

```diff
@@ -438,20 +418,17 @@ if {$openroad_pdn_enable == "true" && \
 # Check Power Network
 ###########################
 
-foreach net [[ord::get_db_block] getNets] {
-  set type [$net getSigType]
-  if {$type == "POWER" || $type == "GROUND"} {
-    set net_name [$net getName]
-    if { ![$net isSpecial] } {
-      utl::warn FLW 1 "$net_name is marked as a supply net, but is not marked as a special net"
-    }
-    if { $openroad_psm_enable == "true" } {
-      puts "Check supply net: $net_name"
-      check_power_grid -net $net_name
-    }
+foreach net [sc_supply_nets] {
+  if { ![[[ord::get_db_block] findNet $net] isSpecial] } {
+    utl::warn FLW 1 "$net_name is marked as a supply net, but is not marked as a special net"
```

Comment:
Just asking because you search for the net after you use `sc_supply_nets` here?

Reply:
OpenROAD uses a hash in findNet so it's relatively fast